app/views/assessor_interface/further_information_requests/edit.html.erb

```diff
@@ -7,8 +7,28 @@
   <h2 class="govuk-heading-m"><%= item.fetch(:heading) %></h2>
   <%= govuk_inset_text(text: item.fetch(:description)) %>
   <%= render(CheckYourAnswersSummary::Component.new(**item.fetch(:check_your_answers))) %>
+  <% if item[:further_information_request_item].work_history_contact? %>
+    <%= govuk_details(summary_text: "Review previous referee details?") do %>
+      <%= govuk_summary_list do |summary_list| %>
+        <%= summary_list.with_row do |row| %>
+          <% row.with_key { "Contact name" } %>
+          <% row.with_value { item[:further_information_request_item].work_history.contact_name } %>
+        <% end %>
+        <%= summary_list.with_row do |row| %>
+          <% row.with_key { "Contact job" } %>
+          <% row.with_value { item[:further_information_request_item].work_history.contact_job } %>
+        <% end %>
+        <%= summary_list.with_row do |row| %>
+          <% row.with_key { "Contact email" } %>
+          <% row.with_value { item[:further_information_request_item].work_history.contact_email } %>
+        <% end %>
+      <%end %>
+    <% end %>
+  <% end %>
```

Comment:
I wonder if rather than passing the `further_information_request_item` here in the `review_items` to the view could we instead pass the rows for this summary list. We use the [view object pattern](https://www.nopio.com/blog/view-objects-rails-views/) in this project to keep the views nice and small, and since the `review_items` method is coming from a view object I think we should push more of the logic in to the view object.

Here's an example of how to use the component by passing in the data directory as a hash: https://govuk-components.netlify.app/components/summary-list/#building-a-summary-list-directly-from-data